umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif9.clj

```diff
@@ -118,7 +125,9 @@
           [:Epoch Epoch]
           [:Stage Stage]
           [:Detailed_Classification DetailedClassification]])])
-    [:Data_Set_Progress (:CollectionProgress c)]
+  (when-let [c-progress (when-let [coll-progress (:CollectionProgress c)]
```

Comment:
I actually did that before, but I saw some old code did it this way, so I changed it.